index.js

```diff
@@ -688,25 +688,38 @@ prototype.buildModel = function(oldModel) {
   var properties = {};
 
   this.forEach(oldModel.properties, function(oldProperty, propertyName) {
+    var allowedProperties = {
+      description: oldProperty.description
+    };
+
     if (fixNonStringValue(oldProperty.required) === true) {
       required.push(propertyName);
     }
 
+    if (oldProperty.hasOwnProperty('example')) {
+      allowedProperties.example = oldProperty.example;
+    }
+
     properties[propertyName] = extend({},
       this.buildDataType(oldProperty, true),
-      {description: oldProperty.description}
+      allowedProperties
```

Comment:
You can see check here https://github.com/apigee-127/swagger-converter/blob/master/index.js#L824